completions/bash/oci-runtime-tool

```diff
@@ -130,11 +130,9 @@ __oci-runtime-tool_complete_compliance_level() {
 __oci-runtime-tool_complete_propagations() {
 	COMPREPLY=( $( compgen -W "
 		private
-		rprivate
-		rshared
-		rslave
```

Comment:
This catches up with the 1.0 spec, but we may want to see how opencontainers/runtime-spec#923 plays out before removing them.  Or we can remove them now and restore them later if the the `r*` forms get added to the spec.

Reply:
I think we can remove them now, and make changes when spec changes.

Reply:
+1, we can remove them now.